addons/web/static/tests/views/view_dialogs/select_create_dialog.test.js

```diff
@@ -349,8 +349,12 @@ test("SelectCreateDialog list view is readonly", async () => {
     });
     await animationFrame();
 
-    // click on the first row to see if the list is editable
+    // click on the first row twice to see if the list is editable
```

Comment:
this test is a bit stupid, because the first td is the cell of the record selector, and clicking on it never toggles the edit mode, even in editable lists ; it always selects the row

Reply:
I'm wondering whether it's still relevant with the recent changes on selection. Before, you could still click on a not yet selected records (while other records were selected) to toggle the edit mode. Now you can't anymore. So I don't see how we could switch a row in edition in a SelectCreateDialog with the current state of the codebase.